Library/Homebrew/cmd/update-report.rb

```diff
@@ -235,22 +236,25 @@ def output_update_report
       EOS
     end
 
-    return if new_repository_version.blank?
+    return if new_tag.blank? || args.quiet?
 
     puts
-    ohai "Homebrew was updated to version #{new_repository_version}"
-    Settings.write "latesttag", new_repository_version
-    if new_repository_version.split(".").last == "0"
+    ohai "Homebrew was updated to version #{new_tag}"
```

Comment:
```suggestion
```

I realise this was already here but I think it can be removed given the messages above and tweaks to messages below. Thoughts?

Reply:
I think this (probably with the empty `puts` above) should stay.

The original point of showing the changelogs and stuff is to bring deprecations and changes to people's attention, so bold is nice for that.

Also, there can be a big gap between the above message and this since all of the formula changes (including, sometimes, the message about outdated formuale/casks) and tap fetch errors all show up in between.

Reply:
I agree that the highlighted line is redundant. I removed it and added `ohai` to earlier message.